src/java/org/apache/cassandra/db/partitions/BTreePartitionUpdate.java

```diff
@@ -249,6 +266,19 @@ public int operationCount()
              + (deletionInfo.getPartitionDeletion().isLive() ? 0 : 1);
     }
 
+    // FIXME review
+    /**
+     * The accumulated BTree size of the data contained in this update.
+     *
+     * @return the accumulated BTree size of the data contained in this update.
+     */
+    @VisibleForTesting
+    @Override
+    public long accumulatedDataSize()
+    {
+        return BTree.<Row>accumulate(holder.tree, (row, value) -> row.dataSize() + value, 0L);
+    }
+
```

Comment:
For Review:
`accumulatedDataSize` and `unsharedHeapSize` are coming from 5.0 before `PartitionUpdate` became an interface. The implementation has moved to `BTreePartitionUpdate` and `TriePartitionUpdate`.